test/App.test.jsx

```diff
@@ -0,0 +1,21 @@
+import { fireEvent, render } from '@testing-library/react';
+import App from '../src/App';
```

Comment:
```suggestion
import { fireEvent, render } from '@testing-library/react';

import App from '../src/App';
```

의미상으로 구분되는 경우 빈 줄을 넣어서 구분해주면 좋아요.